templates/base.html

```diff
@@ -65,7 +65,10 @@
     {{template "body" .}}
     <!-- Placed at the end of the document so the pages load faster -->
     <script src="/js/dist/vendor.min.js"></script>
-    <script src="/js/dist/app/gophish.min.js"></script>
+
+    <!-- TODO - use "min" -->
```

Comment:
Before merging, we should change the reference to `min.js`, but please don't actually commit your copy of `.min.js`- I'll compile that myself and commit it after merging these changes.